emails/views.py

```diff
@@ -363,14 +363,13 @@ def _get_attachment_metrics(part):
         [attachment_extension_tag, attachment_content_type_tag]
     )
 
-    attachment = tempfile.NamedTemporaryFile(
+    attachment = SpooledTemporaryFile(
+        max_size=150*1000,  # 150KB max from SES
```

Comment:
Does the 150KB max for SES apply to each attachment? Or to the entire email contents/data? We can probably still merge & deploy this to start - so we can get SOME attachment support now. But we might need to consider what the best `max_size` for the attachments should be, based on real-world measurements.

Reply:
150KB max is for entire email. So yeah this limit is probably not very useful right now and we will want to revisit this when we move to S3 and we have a bigger email size limit.